charts/kof-collectors/values.yaml

```
@@ -217,6 +217,19 @@ opentelemetry-kube-stack:
             send_batch_max_size: 1500
             send_batch_size: 1000
             timeout: 1s
+
+          transform/syslogkvparser:
+            error_mode: silent
+            log_statements:
+              - context: log
+                conditions:
+                  - attributes["log.file.path"] == "/hostfs/var/log/syslog"
+                  - attributes["process.executable.name"] == "k0s"
+                statements:
+                  - set(log.cache["kv_raw"], log.body)
+                  - replace_pattern(log.cache["kv_raw"], ".*?(\\w+=)(?:\\\\?\"(.*?)\\\\?\"|(\\S+)).*?|.+", "$1$2$3|")
+                  - replace_pattern(log.cache["kv_raw"], "\\|+", "|")
+                  - merge_maps(log.attributes, ParseKeyValue(log.cache["kv_raw"], "=", "|"), "upsert") where cache["kv_raw"] != "|"
```

Not sure, but maybe it's better to use `insert` [strategy](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/pkg/ottl/ottlfuncs/README.md#merge_maps) instead of `upsert` to avoid unexpectedly overwriting existing attributes that came from more reliable sources like `k0s` above.
```suggestion
                  - merge_maps(log.attributes, ParseKeyValue(log.cache["kv_raw"], "=", "|"), "insert") where cache["kv_raw"] != "|"
```